docs/source/cleanlab/datalab/guide/issue_type_description.rst

```diff
@@ -107,6 +109,15 @@ The assumption that examples in a dataset are Independent and Identically Distri
 
 For datasets with low non-IID score, you should consider why your data are not IID and act accordingly. For example, if the data distribution is drifting over time, consider employing a time-based train/test split instead of a random partition.  Note that shuffling the data ahead of time will ensure a good non-IID score, but this is not always a fix to the underlying problem (e.g. future deployment data may stem from a different distribution, or you may overlook the fact that examples influence each other). We thus recommend **not** shuffling your data to be able to diagnose this issue if it exists.
 
+
+Image-specific Issues
+---------------------
+
+For image datasets which are properly specified as such, Datalab can detect additional types of image-specific issues (if the necessary optional dependencies are installed).
+Specifically, low-quality images which are too: dark/bright, blurry, low information, abnormally sized, etc.
+Descriptions of these image-specific issues are provided in the `CleanVision package <https://github.com/cleanlab/cleanvision>`_ and its documentation.
+
+
 Optional Issue Parameters
 =========================
 
```

Comment:
consider whether we should have an Image-Specific Issue Parameters section in the “Optional Issue Parameters” guide which links to how somebody can specify cleanvision args via datalab.
[https://github.com/sanjanag/cleanlab/blob/cleanvision-integration/docs/source/cleanlab/datalab/guide/issue_type_description.rst#optional[…]arameters](https://github.com/sanjanag/cleanlab/blob/cleanvision-integration/docs/source/cleanlab/datalab/guide/issue_type_description.rst#optional-issue-parameters)

Reply:
cc @elisno